frontend/package.json

```diff
@@ -0,0 +1,77 @@
+{
+    "name": "whiteboard-app",
+    "version": "1.0.0",
+    "author": "skusnierz",
+    "dependencies": {
+        "@material-ui/core": "^4.11.3",
+        "@material-ui/icons": "^4.11.2",
+        "@material-ui/lab": "^4.0.0-alpha.57",
+        "@testing-library/jest-dom": "^5.11.4",
+        "@testing-library/react": "^11.1.0",
+        "@testing-library/user-event": "^12.1.10",
+        "@types/jest": "^26.0.15",
+        "@types/node": "^12.0.0",
+        "@types/react": "^16.9.53",
+        "@types/react-dom": "^16.9.8",
+        "axios": "^0.21.1",
+        "body-parser": "^1.19.0",
+        "cors": "^2.8.5",
+        "js-cookie": "^2.2.1",
+        "moment": "^2.29.1",
```

Comment:
You probably don't want to use moment. This library is sooo heavy and you don't need 99% of its functionality.